devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp

```diff
@@ -572,5 +768,21 @@ bool YarpRobotLoggerDevice::detachAll()
 
 bool YarpRobotLoggerDevice::close()
 {
+    // stop all the video thread
```

Comment:
should we add a mutex here? or is it not necessary?

Reply:
`VideoWriter::recordVideoIsRunning` is an atomic bool so the mutex is not necessary